yarn/chapter.md

```diff
@@ -374,7 +374,8 @@ When fixes for these bugs are merged into master, they are added to the version
 After that, the changes are added to npm, and the CI builds and deploys the new version.
 
 #### Feature Flags
-<div id="feature-flags" />
+<div id="feature-flags"></div>
```

Comment:
Actually, there should be no need for this `<div>` in general. Headings should have a linkable ID already based on the heading (e.g. https://delftswa.gitbooks.io/desosa-2017/content/yarn/chapter.html#feature-flags for the Feature Flags section). The end-result is the same though. Want me to merge this fix or do you want to change to the native method instead?